pkg/karmadactl/get/get.go

```diff
@@ -61,6 +61,17 @@ const (
 	proxyURL              = "/apis/cluster.karmada.io/v1alpha1/clusters/%s/proxy/"
 )
 
+type adoption string
+
+const (
+	// managedByKarmada indicate that these are resources of member clusters and are managed by the Karmada control plane.
+	managedByKarmada adoption = "Y"
+	// notManagedByKaramda indicate that these are resources of member clusters and are not managed by the Karmada control plane.
+	notManagedByKaramda adoption = "N"
+	// none indicate that these are Karmada control plane resources.
+	none adoption = "-"
```

Comment:
For those resources from the Karmada plane, use `-` or `N/A`?
Any experience we can get from kubectl?

Reply:
There are no similar cases found from kubectl. Using "-" might be more succinct compared to "N/A".

Reply:
OK.